lib/optimize/SplitChunksPlugin.js

```diff
@@ -105,8 +105,8 @@ module.exports = class SplitChunksPlugin {
 			),
 			minSize: options.minSize || 0,
 			minChunks: options.minChunks || 1,
-			maxAsyncRequests: options.maxAsyncRequests || 1,
-			maxInitialRequests: options.maxInitialRequests || 1,
+			maxAsyncRequests: options.maxAsyncRequests || 6,
+			maxInitialRequests: options.maxInitialRequests || 4,
```

Comment:
Not sure if the above changes are needed since we already do that in `WebpackOptionsDefaulter`.

/cc @ooflorent

Reply:
No. Leave them untouched.